app/controllers/journeys/adjustments/profitOrLoss/ProfitOrLossCalculationController.scala

```diff
@@ -42,12 +44,26 @@ class ProfitOrLossCalculationController @Inject() (override val messagesApi: Mes
     with Logging {
 
   def onPageLoad(taxYear: TaxYear, businessId: BusinessId): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    val summaryList = SummaryListCYA.summaryListOpt(List())
+    val summaryList               = SummaryListCYA.summaryListOpt(List())
+    val netAmount                 = BigDecimal(4600)
+    val formattedNetAmount        = formatSumMoneyNoNegative(List(netAmount))
+    val yourAdjustedProfitTable   = buildTable1(taxYear)
+    val netProfitTable            = buildTable2()
+    val additionsToNetProfitTable = buildTable3()
+    val capitalAllowanceTable     = buildTable4()
+    val adjustmentsTable          = buildTable5()
```

Comment:
Do these changes need to also be represented in the ProfitOrLossCalculationControllerSpec?